facade/nodeFacade.go

```diff
@@ -336,7 +339,10 @@ func (nf *nodeFacade) ComputeTransactionGasLimit(tx *transaction.Transaction) (*
 
 // GetAccount returns a response containing information about the account correlated with provided address
 func (nf *nodeFacade) GetAccount(address string, options apiData.AccountQueryOptions) (apiData.AccountResponse, apiData.BlockInfo, error) {
-	accountResponse, blockInfo, err := nf.node.GetAccount(address, options)
+	ctx, cancel := nf.getContextForApiTrieRangeOperations()
```

Comment:
could we do the getContextForApiTrieRangeOperations only if withKeys option is set?

Reply:
yes, changed it

Reply:
good observation: refactored to have GetAccountWithKeys